pkg/instance/repository.go

```diff
@@ -164,6 +164,16 @@ func (r repository) FindDeployments(groupNames []string) ([]*model.Deployment, e
 	return deployments, err
 }
 
+func (r repository) FindPublicDeployments() ([]*model.Deployment, error) {
```

Comment:
Not without changing code that we're also using when returning deployments elsewhere. I'd like to have this PR merged since the frontenders depend on it. So I'd appreciate keeping it as simple as possible.